doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -8,13 +8,13 @@ The Anuket Kubernetes Reference Architecture (RA) is intended to be an industry
 standard-independent Kubernetes reference architecture that is not tied to any
 specific offering or distribution. No vendor-specific enhancements are required
 to achieve conformance with the Anuket specifications.
-Conformance is achieved by using upstream components or features that are
-developed by the open source community. This allows operators to have a common
-Kubernetes-based architecture that supports any conformant VNF or CNF deployed on
-it to operate as expected. The purpose of this chapter is to outline all the
-components required to provide Kubernetes in a consistent and reliable way. The
-specification of how to use these components is detailed in
-Chapter 04 :ref:`chapters/chapter04:component level architecture`.
+Conformance to these specifications can be ensured by successfully running RC2
+and by using upstream components or features that are developed by the open source
```

Comment:
This is still confusing. Can this sentence be interpreted as saying that just by "using features developed by the open-source community" an implementation is RA2 conformant?

May I suggest:
Conformance to these specifications can be ensured by: 
1. successfully running RC2 tests, and
2. using upstream components or features that are developed by the open-source community.

What is a conformant VNF or CNF?

I don't think "allows" is appropriate here. Should it be, "Using the RA2 Kubernetes-based architecture specifications, operators can deploy infrastructure that will run any  VNF or CNF that has successfully run on an RA2 conformant infrastructure."

Reply:
Fixed in [84d7354](https://github.com/cntt-n/CNTT/pull/3217/commits/84d73542c02ac1e2ea40d950c156f355dbf2d3a9)